Doc/library/string.rst

```diff
@@ -571,6 +571,12 @@ The available presentation types for :class:`float` and
    |         | the current locale setting to insert the appropriate     |
    |         | number separator characters.                             |
    +---------+----------------------------------------------------------+
+   | ``'x'`` | For :class:`float`, hex format. Outputs the number in    |
+   |         | base 16, using lower-case letters for the digits above 9.|
+   |         | The alternate form adds ``'0x'`` prefix.                 |
```

Comment:
```suggestion
   | ``'x'`` | For :class:`float`, hex format, supported by             |
   |         | :meth:`~float.fromhex`.   Represent the number by a      |
   |         | hexadecimal string in the form ``[±][0x]h[.hhh]p±d``,    |
   |         | using using lower-case letters for the digits above 9.   |
   |         | The exponent ``d`` is written in decimal, it always      |
   |         | contains at least one digit, and it gives the power of 2 |
   |         | by which to multiply the coefficient.                    |
   |         |                                                          |
   |         | The alternate form adds ``'0x'`` prefix.                 |
```

1) maybe the first sentence + remark about lower-case is enough
2) maybe we should document also 1-normalisation & case of subnormals.